modules/DMZ/data_kit/filler_strategy.py

```diff
@@ -36,6 +34,7 @@ def get_missing_ratios_dict(self):
         missing_ratios = dataset_insight.get_missing_ratios(self.pandas_dataset, "column")
         self.missing_ratios_dict = dict(zip(self.pandas_dataset.columns.values, missing_ratios))
 
+
     def get_to_do_list(self):
```

Comment:
I wrote it, I'll add some documentation to it next time I make a PR. I'll be doing a similar strategy pattern soon.